auslib/web/admin/swagger/api.yaml

```diff
@@ -108,6 +131,110 @@ parameters:
     minimum: 0
     required: true
 
+  change_id_param:
+    name: change_id
+    in: path
+    description: Unique id to identify the identify the change
+    type: string
+    required: true
+
+  field_name_param:
+    name: field
+    in: path
+    description: name of DB column whose history is requested
+    type: string
+    required: true
+
+  userPermissionFormDataParam:
+    name: user_permission_request_body
+    in: body
+    description: Request body containing options and data version field values for manipulating user permission values in DB
+    required: true
+    schema:
+      allOf:
+        - $ref: '#/definitions/CSRFModel'
+        - type: object
+          properties:
+            data_version:
+              type: integer
+              format: int32
+              description: data_version of permission
+              minimum: 1
+              example: 1
+            options:
+              type: ["string", "null"]
+              format: JSONStringField
+              description: JSON field specifying the actions that user can perform and the prdouct on which it has current permission to do so
+              example: "{'actions':['modify']}"
+          # accomodate 'options_as_json' and 'permission' which are duplicate values of 'options'
+          # and 'permission path parameter' respectively, getting passed in request body
+          # No need to validate the duplicate values again
+          additionalProperties: true
```

Comment:
I was unable to debug the issue on UI. AFAIK , the duplicate value is getting set [here](https://github.com/mozilla/balrog/blob/master/ui/app/js/controllers/user_edit_controller.js#L249) but in next line the API is invoked. Also here's the definition of permission i.e. data object. [link](https://github.com/mozilla/balrog/blob/master/ui/app/js/controllers/user_edit_controller.js#L139) , it defines options_as_json and permission so that they can be displayed on the UI after adding or updating the permissions. Example: [link](https://github.com/mozilla/balrog/blob/master/ui/app/templates/permissions_modal.html#L52) , [link2](https://github.com/mozilla/balrog/blob/master/ui/app/templates/permissions_modal.html#L56). Not sure how to fix this issue on the UI